### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by rahuls-microsoft**
Adding @rahul-microsoft for fleet

# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Comment by rahuls-microsoft** [comment]
Adding my Github alias to CODEOWNERS.
@jsquire - Is this the correct process? I have also added myself in the PR - https://github.com/Azure/azure-sdk-for-net/pull/46541

Please let me know the right process to add.

**Comment by jsquire** [comment]
> Adding my Github alias to CODEOWNERS. @jsquire - Is this the correct process? I have also added myself in the PR - #46541
> 
> Please let me know the right process to add.

Please see the links that I left above.   Adding yourself to CODEOWNERS does not fix your memberships.  You need to make your Azure membership public.

**Comment by rahuls-microsoft** [comment]
image

Made the Azure profile public.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1070,11 +1070,9 @@
 # ServiceOwners:                                                   @sunkun99
 
 # PRLabel: %Compute - Fleet
-/sdk/computefleet/                                                 @sahilarora92 @dhruvil009
-
+/sdk/computefleet/                                                 @sahilarora92 @dhruvil009 @rahuls-microsoft
 # ServiceLabel: %Compute - Fleet
-# ServiceOwners:                                                   @sahilarora92 @dhruvil009
-
+# ServiceOwners:                                                   @sahilarora92 @dhruvil009 @rahuls-microsoft
```
Comment (jsquire) [inline .github/CODEOWNERS:1075]:
```suggestion
# ServiceOwners:                                                   @sahilarora92 @dhruvil009 @rahuls-microsoft

```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1070,10 +1070,9 @@
 # ServiceOwners:                                                   @sunkun99
 
 # PRLabel: %Compute - Fleet
-/sdk/computefleet/                                                 @sahilarora92 @dhruvil009
-
+/sdk/computefleet/                                                 @sahilarora92 @dhruvil009 @rahuls-microsoft
 # ServiceLabel: %Compute - Fleet
```
Comment (jsquire) [inline .github/CODEOWNERS:1074]:
```suggestion
# ServiceLabel: %Compute - Fleet %Mgmt
```

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1070,10 +1070,9 @@
 # ServiceOwners:                                                   @sunkun99
 
 # PRLabel: %Compute - Fleet
-/sdk/computefleet/                                                 @sahilarora92 @dhruvil009
-
-# ServiceLabel: %Compute - Fleet
-# ServiceOwners:                                                   @sahilarora92 @dhruvil009
+/sdk/computefleet/                                                 @sahilarora92 @dhruvil009 @rahuls-microsoft
```
Comment (jsquire) [inline .github/CODEOWNERS:1073]:
```suggestion
/sdk/computefleet/Azure.ResourceManager.*/                         @sahilarora92 @dhruvil009 @rahuls-microsoft
```

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1070,10 +1070,9 @@
 # ServiceOwners:                                                   @sunkun99
 
 # PRLabel: %Compute - Fleet
-/sdk/computefleet/                                                 @sahilarora92 @dhruvil009
-
-# ServiceLabel: %Compute - Fleet
-# ServiceOwners:                                                   @sahilarora92 @dhruvil009
+/sdk/computefleet/Azure.ResourceManager.*/                         @sahilarora92 @dhruvil009 @rahuls-microsoft
+# ServiceLabel: %Compute - Fleet %Mgmt
```
Comment (jsquire) [inline .github/CODEOWNERS:1074]:
```suggestion

# ServiceLabel: %Compute - Fleet %Mgmt
```

**Review by jsquire — APPROVED** [review]
I fixed the formatting errors and missing tags.   Merging.

**Pull request merged**